ironfish/src/rpc/routes/chain/snapshotChain.ts

```diff
@@ -0,0 +1,101 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import bufio from 'bufio'
+import * as yup from 'yup'
+import { Assert } from '../../../assert'
+import { getBlockSize, writeBlock } from '../../../network/utils/block'
+import { BlockchainUtils } from '../../../utils/blockchain'
+import { ApiNamespace, router } from '../router'
+
+const MAX_BLOCKS_PER_SNAPSHOT_CHUNK = 1000
```

Comment:
I feel like blocks could be super variable in size. Would it make more sense to limit based on bytes? Doesn't seem like an urgent change since hopefully we will have consistent block sizes for the rest of testnet but maybe a follow up change to do

Reply:
I think that's a sensible change. I think it will be easy to add once the main function is included in the node and we see how it's used realistically.